### DISCUSSION TITLE
Why is `'disable_existing_loggers': True` when using Sentry?

### DISCUSSION THREAD

**Discussion by cristobalmackenzie** [cookiecutter/cookiecutter-django#5027] [Q&A]
Why is `'disable_existing_loggers': True` when using Sentry?

We just spent a ton of time debugging celery worker/beat logging issues until we figured out this setting. We started a project with `use_sentry: y` but haven't completed the sentry setup, and couldn't see any logs.

**Comment by browniebroke** [comment]
I don't remember if there was a specific reason, to be honest. Seems to be coming from https://github.com/cookiecutter/cookiecutter-django/pull/1820

Feel free to send us a PR to change it with an quick explanation of why we should chnage it, I'm particularly interested in seeing with an example of logging with before/after.
